src/viewport.cpp

```diff
@@ -1458,6 +1473,8 @@ static void ViewportSortParentSprites(ParentSpriteToSortVector *psdv)
 				*psd3 = *(psd3 - 1);
 			}
 			*psd = temp;
+			if (psd2 > psdMax)
+				psdMax = psd2;
```

Comment:
OpenTTD-style is single-line or use brackets. This is fine here:
`if (psd2 > psdMax) psdMax = psd2;`